cmd/neofs-node/cache.go

```diff
@@ -314,16 +315,21 @@ func (s *ttlContainerLister) List(id *user.ID) ([]cid.ID, error) {
 func (s *ttlContainerLister) update(owner user.ID, cnr cid.ID, add bool) {
 	strOwner := owner.EncodeToString()
 
-	val, ok := (*ttlNetCache)(s).cache.Get(strOwner)
+	val, ok := s.inner.cache.Peek(strOwner)
```

Comment:
what does `Get` -> `Peek` change solve?

Reply:
`Peek` is faster (read-only) and does not update the position of an item in LRU cache -- which is what we do here, we don't _use_ anything, just passively update.